tests/fixtures/noqa/noqa38.py

```diff
@@ -49,3 +49,29 @@ def wrong_comprehension2():
 
 def positional_only(first, /, second):  # noqa: WPS451
     anti_wps428 = 1
+
+
+for element in range(10):
+    try:  # noqa: WPS452
+        my_print(1)
+    except AnyError:
+        my_print('nope')
+    finally:
+        if element == 1:
+            # See:
+            # https://github.com/wemake-services/wemake-python-styleguide/issues/1082
+            continue
+        else:
```

Comment:
`else` can be removed

Reply:
If I remove it there would be an error like `AssertionError: Violation without a #noqa count` (error WPS327). 
Of course I can add it to `try` like `try:  # noqa: WPS452, WPS327` but it also leads adding WPS327 to VERSION_SPECIFIC in test_noqa.py.
Maybe you can advise me how I can delete `else` and `except` statements without adding such violations to VERSION_SPECIFIC?

Reply:
We can try this sample: 

```python
for element in range(10):
    try:  # noqa: WPS452
        my_print(1)
    except AnyError:
        my_print('nope')
    finally:
        continue
    my_print(2)
```